src/presentation/ui/presenter/dto/marketplaceOverview.go

```diff
@@ -0,0 +1,29 @@
+package presenterDto
```

Comment:
Why create a whole layer man? You only use it in one place, put that on the top of the page and it's fine.